cmd/admin-router.go

```diff
@@ -45,9 +45,12 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 	// Get status, update, restart and stop MinIO service.
 	adminV1Router.Methods(http.MethodPost).Path("/service").HandlerFunc(httpTraceAll(adminAPI.ServiceActionHandler)).Queries("action", "{action:.*}")
 
-	// Info operations
+	// Info operations - return details based on input type
 	adminV1Router.Methods(http.MethodGet).Path("/info").HandlerFunc(httpTraceAll(adminAPI.ServerInfoHandler))
 
+	// Server Info operations - return details based on input type
+	adminV1Router.Methods(http.MethodGet).Path("/serviceinfo").HandlerFunc(httpTraceAll(adminAPI.ServiceInfoHandler)).Queries("type", "{serviceType:.*}")
```

Comment:
This can be `Queries("type", "{type:.*}")`. As it is self-inferred that the `type` here means `service type`